scripts/update_teams.py

```diff
@@ -107,9 +83,7 @@ def create_team(org, name, description, repos):
     env = jinja2.Environment(undefined=NullUndefined)
 
     with open(recipe) as fh:
-        environ = SilentDict()
-        environ.undefined = ""
-        contents = env.from_string(''.join(fh)).render(environ=environ)
+        contents = env.from_string(''.join(fh)).render()
```

Comment:
I think all that was missing is we need to make `environ` an empty dictionary. Currently it is just undefined. So, that should fix it. Unfortunately, I don't have the time to play with it now.